src/main/java/com/airsquared/blobsaver/Controller.java

```diff
@@ -657,7 +680,14 @@ public void helpLabelHandler(MouseEvent evt) {
     }
 
     @SuppressWarnings("ResultOfMethodCallIgnored")
-    public void aboutMenuHandler() {
+    public void aboutMenuHandler() { //handles the "About Blobsaver" menu
+        Stage aboutStage;
```

Comment:
I'm confused about the use of this. What does it do?(it is producing the exact same result for me as before)

Reply:
With my new global mac menu bar, it is possible to open multiple about menus. I want to check if there is already an about menu open. If so, I focus on it and not create a new one. 

The `setAlwaysOnTop` makes sure it is always on top, since I was having a problem with it not doing that. 

Even if it's supposed to be always on top anyway, it doesn't hurt to add this.